src/main/java/domains/share_project/share_project_page/ShareProjectPageInteractor.java

```diff
@@ -0,0 +1,33 @@
+package domains.share_project.share_project_page;
+
+import java.util.List;
+public class ShareProjectPageInteractor implements ShareProjectPageInputBoundary {
+   private final ShareProjectPageOutputBoundary presenter;
+   private final ShareProjectPageDataAccessInterface dataAccessInterface;
+   public ShareProjectPageInteractor(
+           ShareProjectPageOutputBoundary presenter,
+           ShareProjectPageDataAccessInterface dataAccessInterface
+   ) {
+      this.presenter = presenter;
+      this.dataAccessInterface = dataAccessInterface;
+   }
+
+   public void execute(ShareProjectPageInputData input) {
+      try {
+         List<List<String>> usersNameAndId = dataAccessInterface.getUsersNameAndId();
+
+         for (List<String> userNameAndId : usersNameAndId) {
```

Comment:
It will be better to filter the people with permission in DataAccessInterface layer, rather than interactor for better performance.